config/controller-service.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Red Hat
```

Comment:
maybe for convention you will want to keep the number based ordering of the file i.e: 100-namespace.yaml etc..